cmd/auth/main.go

```diff
@@ -242,9 +242,14 @@ func main() {
 	webappRouter.Use(auth.MakeMiddleware(authDependency, auth.NewCSRFMiddleware))
 	webappRouter.Use(webapp.PostNoCacheMiddleware)
 
-	webappAnonymousRouter := webappRouter.NewRoute().Subrouter()
-	webappAnonymousRouter.Use(webapp.RequiredAnonymousMiddleware{}.Handle)
-	webapphandler.AttachRootHandler(webappAnonymousRouter, authDependency)
+	webappAuthRouter := webappRouter.NewRoute().Subrouter()
+	webappAuthRouter.Use(webapp.RequiredAnonymousMiddleware{}.Handle)
```

Comment:
Seems not related to this PR, but should we change this middleware name? 🙈

Reply:
What about `RequiredNotAuthenticatedMiddleware`?